conda_smithy/github.py

```diff
@@ -172,7 +172,7 @@ def configure_github_team(meta, gh_repo, org):
     for old_maintainer in current_maintainers_handles - maintainers:
         print(
             "AN OLD MEMBER ({}) NEEDS TO BE REMOVED FROM {}".format(
-                old_maintainer, repo_name
+                old_maintainer, gh_repo
```

Comment:
No, new code. I thought I checked this, but apparently not.

Reply:
This code is run rarely though. (Maintainers leaving happens rarely)

Reply:
Worth checking for sure. The fix is welcome though.

We have gone through a few iterations of this semi-private API in the webservice. So this may be an evolutionary dead end. Would need to check. @isuruf is probably more in touch with what is used currently than I.